EIPS/eip-5792.md

```diff
@@ -2,12 +2,13 @@
 eip: 5792
 title: Wallet Function Call API
 description: Adds JSON-RPC methods for sending multiple function calls from the user's wallet, and checking their status
-author: Moody Salem (@moodysalem)
+author: Moody Salem (@moodysalem), Derek Rein (@arein)
 discussions-to: https://ethereum-magicians.org/t/eip-5792-wallet-abstract-transaction-send-api/11374
-status: Stagnant
+status: Review
 type: Standards Track
 category: Interface
 created: 2022-10-17
+requires: CAIP-25
```

Comment:
Sorry, we don't allow links to CAIPs. If you'd like to propose allowing links to CAIPs, please open a pull request to EIP-1 following the guidelines in [EIP-5757](https://eips.ethereum.org/EIPS/eip-5757)